src/compiler/transformers/add-component-meta-proxy.ts

```diff
@@ -64,7 +64,7 @@ export const createComponentMetadataProxy = (compilerMeta: d.ComponentCompilerMe
  */
 export const createAnonymousClassMetadataProxy = (
   compilerMeta: d.ComponentCompilerMeta,
-  clazz: ts.Expression
+  clazz: ts.ClassExpression
```

Comment:
"clazz" 😂

Reply:
FWIW ☕ - https://stackoverflow.com/questions/2529974/why-do-java-programmers-like-to-name-a-variable-clazz